lnwallet/channel_test.go

```diff
@@ -608,7 +610,9 @@ func TestCheckCommitTxSize(t *testing.T) {
 	checkSize := func(channel *LightningChannel, count int) {
 		// Due to variable size of the signatures (71-73) we may have
 		// an estimation error.
-		BaseCommitmentTxSizeEstimationError := 4
+		// TODO(andrew.shvv) there is additional error, need to
+		// investigate!
+		BaseCommitmentTxSizeEstimationError := 5
```

Comment:
I was able to found that we are generating signature with `70` length, for that reason the error window should be equals to `6` (we have two signatures in witness).